### ISSUE TITLE
hello,a simple question

### ISSUE THREAD

**Issue by puppy2000** [AImageLab-zip/alveolar_canal#9]
I wonder what is the gt that is used to train the Deep Label Expansion stage?From my observation,it is seemd that labels generated from torchio.LabelMap are used.Thanks very much!

**Comment by LucaLumetti** [comment]
Hi @puppy2000, I hope I've understood clearly your question: each patient in the train, test, or eval split has a single `gt_alpha.npy` file which corresponds to its gt. As you can read [in this line](https://github.com/AImageLab-zip/alveolar_canal/blob/190af1c7f17aa5625eccce722e2104497b2f7e69/dataloader/Maxillo.py#L50), this is exactly what it gets loaded and later used as gt. The `torchio.LabelMap` is just a class helpful to warp and handle labels, it does not generate anything, you can find more details on the [Torchio documentation page](https://torchio.readthedocs.io/data/image.html#torchio.LabelMap).
